waiter/src/waiter/state/responder.clj

```diff
@@ -482,9 +483,14 @@
   [{:keys [instance-id->state] :as current-state} update-instance-id->blacklist-expiry-time-fn update-status-tag-fn
    instance-id expiry-time]
   (log/info "unblacklisting instance" instance-id "as blacklist expired at" expiry-time)
-  (cond-> (update-instance-id->blacklist-expiry-time-fn current-state #(dissoc % instance-id))
-    (contains? instance-id->state instance-id)
-    (update-in [:instance-id->state instance-id] update-status-tag-fn #(disj % :blacklisted))))
+  (let [update-instance-id->blacklist-expiry-time-fn-ret
+        (update-instance-id->blacklist-expiry-time-fn current-state #(dissoc % instance-id))]
+    (if (contains? instance-id->state instance-id)
+      (do
+        (scheduler/log-service-instance {:id instance-id} :readmit)
```

Comment:
Did we settle on the `readmit` and `eject` terms?

Reply:
No. We can change them later

Reply:
We definitely had consensus on "eject", but we didn't really discuss the symmetric term.